### DISCUSSION TITLE
Dev server fails to start when build has an error.

### DISCUSSION THREAD

**Discussion by wantingjun** [lowdefy/lowdefy#710] [Questions and Help] [answered]
hi,

I have a question about `dev` command. 
**Scenario:** 
I want to use `npx lowdefy@latest init` to develop project, if one of `- ref` yaml file has error(for example， the format error), the server will not start, it also won't have hot update(I mean, in this situation, even though I fix the wrong yaml file, the server won't start and definitely won't have the web page in browser, the server just keeps in this error state)

the error in terminal is as below:
```
14:08:51 ✖ can not read a block mapping entry; a multiline key may not be an implicit key (6:19)

 3 |     blocks:
 4 |       - id: Context_1626313550088
 5 |         key: '1626313550098'
 6 |         properties:
-----------------------^
 7 |           content: '123'
 8 |           label: {}
```
Then, I fix the yaml file, but the server doesn't work, it‘s still in this error state. I have to terminate the server and re-start it again.

Overall, In start stage, if my yaml file is wrong, after I fix the file,  I expect that the browser should automatically refresh rather than keeping in the error state, so I have to manually restart the server. Could please provide some methods to solve it ?

thank your for your help!!

Best regards，
Tingjun

**Comment by SamTolmay** [comment]
Hi @wantingjun

I have also experienced this issue, and it is quite annoying.

The best thing you can do is to comment out the referenced page that has the issue and then start the server. Then, once the server is running, you can add the page back again, and the server will handle the error like it always does.

I have opened an issue (#711) to fix this behaviour, but unfortunately, I think this will be a lower priority issue, so it might not get resolved soon.

I have also renamed your question so others with the same problem can find it more easily, I hope you don't mind.

**Reply by wantingjun** [reply]
Thank you for your answer!!! hope the issue can be fix by the group.

**Comment by SamTolmay** [comment, marked as answer]
This bug has been resolved in v3.20 😀
